gui/exported.go

```diff
@@ -155,15 +155,13 @@ func (app *App) GoWriteToClipboard(data string) (any, error) {
 }
 
 // App.GoPullLatestNodeConfig pulls the latest node config from the server and returns the network config
-func (app *App) GoPullLatestNodeConfig(network string) (Network, error) {
-	node, err := functions.Pull(network, true)
+func (app *App) GoPullLatestNodeConfig(network string) error {
```

Comment:
Exported wails func can return (data) or (data, err)
sig should be func (app *App) GoPullLatestNodeConfig(network string) (any, err)